pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1859,6 +1859,15 @@ type DMClusterSpec struct {
 	// +optional
 	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
 
+	// ConfigUpdateStrategy determines how the configuration change is applied to the cluster.
+	// UpdateStrategyInPlace will update the ConfigMap of configuration in-place and an extra rolling-update of the
+	// cluster component is needed to reload the configuration change.
+	// UpdateStrategyRollingUpdate will create a new ConfigMap with the new configuration and rolling-update the
+	// related components to use the new ConfigMap, that is, the new configuration will be applied automatically.
+	// +kubebuilder:validation:Enum=InPlace;RollingUpdate
+	// +kubebuilder:default=InPlace
```

Comment:
Please remove this in both master and v1.2. Refer to https://github.com/pingcap/tidb-operator/pull/4317.